includes/statements-functions.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Responsible for statement archive related functionality
+ **/
+
+/**
+ * Class that manages listings of statements and their
+ * filtering options
+ */
+class Statements_View {
+
+	/**
+ 	 * Returns markup for a list of statements.
+	 *
+	 * @since 3.9.0
+	 * @author Jo Dickson
+	 * @return string
+	 */
+	public function get_statements_list_item( $post ) {
+
+		ob_start();
+	?>
+		<?php if ( $post ) : ?>
+			<?php
+				$link         = get_permalink( $post );
+				$title        = get_the_title( $post );
+				$author_id    = get_field( 'post_author_term', $post );
+				$author       = ( $author_id && $author_id->term_id ) ? get_term( $author_id->term_id ) : null;
+				$author_name  = ( $author && $author->name ) ? $author->name : '';
+				$author_title = ( $author ) ? get_field( 'author_title', $author ) : null;
+				$author_title = ( $author_title ) ? ', ' . $author_title : '';
+				$datetime     = get_the_date();
```

Comment:
`get_the_date()` _should_ work as expected when used in The Loop, but since we pass in a `$post` object to `get_statements_list_item()`, we should ideally pass it along to `get_the_date()`: https://developer.wordpress.org/reference/functions/get_the_date/